src/tests/ftest/util/agent_utils_params.py

```diff
@@ -72,6 +72,10 @@ def __init__(self, filename, common_yaml):
         #        Enable client telemetry for all client processes.
         #   - telemetry_retain: <str>, e.g. 5m
         #        Time to retain per-client telemetry data.
+        #   - access_points: <list>, e.g.  ["hostname1:10001"]
```

Comment:
Doesn't leaving `access_points` in the agent config and confusion when the same list is called `mgmt_svc_replicas` in the server and control configs, or can the `access_points` be a subset of the `mgmt_svc_replicas` list?

Reply:
The control config only has a hostlist. A long time ago (pre-1.0), there used to be separate hostlist and access_point lists. Now, the control config just has a hostlist including all of the servers for convenience with operations like storage scan and format.

One of the major drivers for this change is to codify that the `access_points` parameter on the agent does not have to be in sync with the value of `mgmt_svc_replicas` on the server. It just needs to have some server addresses in the list. Doesn't matter which ones. Having the same set as `mgmt_svc_replicas` is a slight optimization, but it will work even if you have none of the replicas in the list.

Down the road, we will likely design a more dynamic MS replica selection similar to how the PS works, and then it will be really clear that the `access_points` list is independent of which servers are the current replicas.